Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.61 AS chef
+WORKDIR app
+
+
+FROM chef AS planner
+COPY src src/
+COPY Cargo.* .
+RUN cargo chef prepare --recipe-path recipe.json
+
+
+FROM chef AS builder
+COPY --from=planner /app/recipe.json recipe.json
+# Build dependencies - separately for Docker layer caching
+RUN cargo chef cook --release --recipe-path recipe.json
+# Build application
+COPY src src/
+COPY Cargo.* .
+RUN cargo build --release --bin ficai-signals-server
+
+
+FROM debian:bullseye-slim AS runtime
```

Comment:
Should we pin this to a specific version like `bullseye-20220509-slim`, or are you expecting that to not matter too much?

Reply:
1) I don't expect it to matter because we just run the executable without any dependencies. But we might benefit from security updates etc
2) cargo-chef version might not really matter, but good catch, pinned it now